src/scripts/coffeemate.test.js

```diff
@@ -14,23 +14,20 @@ describe("coffeemate", () => {
     jest.resetAllMocks();
   });
 
-  it('subscribes to "coffee me" messages and loads the existing queue from the brain', () => {
-    const brainSpy = jest.spyOn(app.brain, "get").mockReturnValue();
-
+  it('subscribes to "coffee me" messages', () => {
     coffeemate(app);
 
     expect(app.message).toHaveBeenCalledWith(
-      /coffee me/i,
+      /coffee me( \S+$)?/i,
       expect.any(Function)
     );
-    expect(app.brain.get).toHaveBeenCalledWith("coffeemate_queue");
-    brainSpy.mockRestore();
   });
 
-  describe("when the coffee queue is initially empty", () => {
+  describe("with an the coffee queue is initially empty", () => {
```

Comment:
hm not sure what this text change is trying to convey

Reply:
I think I got the first two words in a revised description and then got distracted, because it doesn't make sense now... 😭